### PR TITLE
Feat/include support check for ng 11.1.x

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Lumberjack pipeline does not support Angular v11.1.x

Issue Number: N/A

## What is the new behavior?

Lumberjack pipeline supporta Angular v11.1.x

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by SerkanSipahi — COMMENTED** [review]
LGTM.

**Comment by LayZeeDK** [comment]
Try finding the *Approve* button under *Files*, @SerkanSipahi 😊

**Review by SerkanSipahi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -26,51 +26,51 @@
...
     "jasmine-core": "~3.6.0",
-    "jasmine-spec-reporter": "~5.0.2",
+    "jasmine-spec-reporter": "~6.0.0",
     "json": "^10.0.0",
-    "karma": "~5.1.0",
+    "karma": "~6.0.1",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage": "~2.0.3",
     "karma-jasmine": "~4.0.1",
     "karma-jasmine-html-reporter": "^1.5.4",
```
Comment (LayZeeDK) [inline package.json:63]: Did you check for updates to these dependencies?

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (NachoVazquez) [reply package.json:63]: They all should be at the latest version

**Pull request merged**